transcom/mymove
# INT B-20810 GSR move history log

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20810)

## Summary

Now that we have the GSR work knocked out, we can add the implementation of seeing it in the audit log/move history.

Since we weren't tracking the evaluation reports or anything, I had to add a migration file to let the audit history table listen for changes to the `gsr_appeals` table.

This PR does the following:
- adds migration to add `gsr_appeals` action to the `audit_history` table
- updates the `move_history_fetcher.sql` file to pull data from the `gsr_appeals` table and related data needed for the history log
- adds event templates for both GSR appeal actions (violations & serious incidents)

### How to test

1. You'll need to ensure the GSR flag is on in your `.envrc`
2. Log in as a QAE and create an evaluation report with violations and a serious incident
3. Log in as a GSR and leave some appeals on everything - APPEAL EVERYTHING
4. Go in the move history tab and you should see all the crap you appealed

## Screenshots
<img width="1685" alt="Screenshot 2024-11-08 at 6 23 46 PM" src="https://github.com/user-attachments/assets/e2675638-7ca1-482d-81cc-2c5eb3daa6d0">
